libraries/AP_HAL_PX4/RCOutput.cpp

```diff
@@ -101,7 +101,9 @@ void PX4RCOutput::_init_alt_channels(void)
 void PX4RCOutput::set_freq_fd(int fd, uint32_t chmask, uint16_t freq_hz, uint32_t &rate_mask) 
 {
     if (_output_mode == MODE_PWM_BRUSHED16KHZ) {
```

Comment:
It shouldn't be _output_mode >= MODE_PWM_BRUSHED16KHZ to allow futur MODE_PWM_BRUSHED ?